scripts/utils/log.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+# @license
+# This file is part of Bott.
+#
+# This project is dual-licensed:
+# - Non-commercial use: AGPLv3 (see LICENSE file for full text).
+# - Commercial use: Proprietary License (contact D@nielLaCos.se for details).
+#
+# Copyright (C) 2025 DanielLaCos.se
+
+# Logging utilities for Bott scripts
+# Aligned with application logger levels (debug, info, warn, error)
+
+# Color codes for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+# Log file configuration
+# Use process-specific log file to avoid race conditions between concurrent processes
+LOG_FILE="${LOG_FILE:-/tmp/bott-script-$$.log}"
```

Comment:
i wonder if we can pull this from the constants.ts somehow

also rename LOG -> LOGGER (see constants refactor)